test/dlc_tests/ec2/pytorch/inference/test_pytorch_inference.py

```diff
@@ -95,13 +98,15 @@ def test_ec2_pytorch_inference_gpu(
 
 
 @pytest.mark.model("densenet")
+@pytest.mark.deep_canary
```

Comment:
For deep canaries, it may be worth documenting why you chose these particular tests, either in the marker itself or in the test function. If you include the text in the marker, you should be able to programmatically validate that there is a description associated with each marker.

Reply:
I'm just choosing any single test as a "placeholder", such that the test has as little complexity as possible, and is unlikely to be flaky.